src/Symfony/Component/Validator/Tests/Constraints/PwnedValidatorTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\Luhn;
+use Symfony\Component\Validator\Constraints\Pwned;
+use Symfony\Component\Validator\Constraints\PwnedValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PwnedValidatorTest extends ConstraintValidatorTestCase
+{
+    private const RETURN = array(
+        '35E033023A46402F94CFB4F654C5BFE44A1:1',
+        '35F079CECCC31812288257CD770AA7968D7:53',
+        '36039744C253F9B2A4E90CBEDB02EBFB82D:5', // this is the matching line, password: maman
+        '3686792BBC66A72D40D928ED15621124CFE:7',
+        '36EEC709091B810AA240179A44317ED415C:2',
+    );
+
+    protected function createValidator()
+    {
+        $httpClient = function (string $url) {
+            if ('https://api.pwnedpasswords.com/range/3EF27' === $url) {
```

Comment:
String instead of constant? Just asking. :)
private const API_ERROR_VALIDATOR_STRING or sth like that, maybe?